docs/json_data/sysfs_attributes_drm.json

```diff
@@ -539,6 +539,138 @@
             "units": "none",
             "writeable": false,
             "alias": ""
+        },
+        "ACT_FREQ": {
+            "attribute": "freq0/act_freq",
+            "scalar": 1e6,
+            "description": "Actual GPU frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": false,
+            "alias": ""
+        },
+        "CUR_FREQ": {
+            "attribute": "freq0/cur_freq",
+            "scalar": 1e6,
+            "description": "Current GPU frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": false,
+            "alias": "GPU_CORE_FREQUENCY_STATUS"
```

Comment:
Similar question as in i915. This signal indicates the driver's cached frequency expectation. Do we want to alias act_freq instead?